backend/services/interfaces/testSessionService.ts

```diff
@@ -41,33 +41,24 @@ export interface TestSessionResponseDTO {
    * the result of the test session
    * there should be one entry here per student
    * */
-  results?: ResultResponseDTO[];
+  results?: ResultDTO[];
   /** the code that students can use to access the test when it is live */
   accessCode: string;
   /** the time when the test session is started by teacher */
   startTime: Date;
 }
 
-export interface ResultRequestDTO {
+export interface NewResultDTO {
```

Comment:
What are your thoughts on calling this something like `UnscoredResultDTO` or `UnmarkedResultDTO` or something? To include a bit more context in naming & make it a bit more intuitive/descriptive. Or something else if you can think of a better name than those LOL.

Reply:
Let's also add some documentation for the interface, explaining why its necessary as well (i.e what we talked about on call last work session).